docs/warnings/dont-call-proptypes.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Don't Call PropTypes Warning
+layout: single
+---
+
+In a future major release of React, the code that implements PropType validation functions will be stripped in production. Once this happens, any code that calls these functions manually (that isn't stripped in production) will throw an error.
+
+### Declaring PropTypes is still fine
+
+The normal usage of PropTypes is still supported:
+
+```javascript
+Button.propTypes = {
+  highlighted: React.PropTypes.bool
+};
+```
+
+Nothing changes here.
+
+### Don’t call PropTypes directly
+
+Using PropTypes in any other way than annotating React components with them is no longer supported:
+
+```javascript
+var apiShape = React.PropTypes.shape({
+  body: React.PropTypes.object,
+  statusCode: React.PropTypes.number.isRequired
+}).isRequired;
+
+// Not supported!
+var error = apiShape(json, 'response');
+```
+
+If you depend on using PropTypes like this, we encourage you to use or create a fork of PropTypes (such as [these](https://github.com/aackerman/PropTypes) [two](https://github.com/developit/proptypes) packages).
```

Comment:
I thought we talked about potentially splitting the PropTypes out from the main package and shipping it as a standalone thing. But I guess that's tangential and not worth mentioning until/if we do it. Just ignore me.